android/src/test/java/vn/luongvo/kmm/survey/android/ui/screens/login/LoginScreenTest.kt

```diff
@@ -103,10 +104,12 @@ class LoginScreenTest {
 
     private fun initComposable(testBody: ComposeContentTestRule.() -> Unit) {
         composeRule.setContent {
-            LoginScreen(
-                viewModel = viewModel,
-                navigator = { destination -> expectedAppDestination = destination }
-            )
+            ComposeTheme {
+                LoginScreen(
+                    viewModel = viewModel,
+                    navigator = { destination -> expectedAppDestination = destination }
```

Comment:
This is a bit long, maybe add line break?

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/63/commits/94327c04b190ec7eb2d57794c1ee961cfc7afbda